common/app/conf/switches/FeatureSwitches.scala

```diff
@@ -506,4 +506,14 @@ trait FeatureSwitches {
     sellByDate = never,
     exposeClientSide = true
   )
+
+  val SubscribeWithGoogle = Switch(
+    SwitchGroup.Feature,
+    "subscribe-with-google",
+    "If switched on, a Subscribe with Google button will appear on AMP articles.",
+    owners = Seq(Owner.withName("adem.gaygusuz")),
+    safeState = Off,
+    sellByDate = never,
```

Comment:
Can you initially give it a sell-by-date? As it's a new feature, which is untested and unproven it feels more appropriate to have a forced review after a few months say.

Reply:
Done, set to `new LocalDate(2019, 3, 29)`. The finish date of the integration project.. also the 13th Friday in 2019.. oh and Brexit day  😲

Reply:
ha sorry to be annoying but could we pick either the week before or after? Brexit day (if it happens) would probably be one of the biggest traffic days ever so keen to avoid switches breaking the build on that day (which is what happens when a switch expires).

Reply:
Good job I mentioned that. I've set it to Wednesday 20/3/19 so that it's mid week, Also, we'll still be here, all being well.